storage/persistence.go

```diff
@@ -59,3 +66,28 @@ func (b *Bucket) InsertLogChunks(ctx context.Context, buildID string, testID str
 
 	return nil
 }
+
+// CheckMetadata returns whether the metadata file exists for the given build
+// and (optional) test.
+func (b *Bucket) CheckMetadata(ctx context.Context, buildID string, testID string) (bool, error) {
```

Comment:
I know it's hard to test views.go, but we should probably at least have a small test for this function.

Reply:
Good catch, added tests!